.circleci/config.yml

```diff
@@ -222,7 +222,10 @@ workflows:
           name: "linkcheck-plugin"
           image-name: "cimg/openjdk:11.0.19"
           command: "./.ci/run-link-check-plugin.sh"
-
+      - validate-with-maven-script:
+          name: "linkcheck-plugin-EXTERNAL"
+          image-name: "cimg/openjdk:11.0.19"
+          command: "./.ci/run-link-check-plugin.sh --check-external"
```

Comment:
We should not do this in Circle CI.

GitHub Action is master push based, it run full validation of links, good. But circleci ok to skip checking external. It will be stable if we skip doing it.

Chance that we leak in PR external link that doesn't work is low.

Ok to keep only to test that it works as separate commit for removal before merge

Reply:
Ok this is what is done already.

Reply:
I keep it unresolved to show how it was tested